integration/commands_administration_test.go

```diff
@@ -159,24 +159,138 @@ func TestCommandsAdministrationListDatabases(t *testing.T) {
 	db := collection.Database()
 	name := db.Name()
 
-	actual, err := db.Client().ListDatabases(ctx, bson.D{{"name", name}})
-	require.NoError(t, err)
-	require.Len(t, actual.Databases, 1)
+	testCases := []struct { //nolint:vet // for readability
```

Comment:
I'd suggest to make a map here to stay consistent with our other integration tests (such as this https://github.com/FerretDB/FerretDB/blob/main/integration/create_test.go#L252-L259)

Reply:
Sure! Changed to a map.